### ISSUE TITLE
LND configuration chapter small issue

### ISSUE THREAD

**Issue by derrickdeek** [Stadicus/RaspiBolt#748]
A small issue while trying to set up lnd.conf
On line 10 of the lnd.conf I got any error:
/home/bitcoin/.lnd/lnd.conf:10: unknown option: protocol.anchors
When checking sample lnd conf files it seems that you need to use protocol.**no-anchors**=true
Then lnd can configure correctly

**Comment by VajraOfIndra** [comment]
Yes, there is a mismatch between the LND version in the guide (v.0.11) and the mentionned anchor option which afaik only works for v.0.12.

I'm working on a PR that updates the guide to the latest v0.13 version which has anchor channels by default (so the anchor channel option can be deleted altogether, unless one does NOT want to use anchor channels, and in that case would need to use the no-anchors option that you mention - but I believe leaving the default is best for the guide, right?)

**Comment by xanoni** [comment]
I actually sent a tiny PR to `lnd` for this a month ago, which was accepted .. see here .. https://github.com/lightningnetwork/lnd/pull/5439 

TL;DR: the new setting is called `protocol.no-anchors=false` (default)
